yaml/com/ibm/VPD/FRUManager.interface.yaml

```diff
@@ -0,0 +1,122 @@
+description: >
+    Implement to manage VPD in the system.
+
+methods:
+    - name: WriteKeyword
+      description: >
+          Method to update the keyword value with the given value. Updates the
+          keyword on hardware and also updates on the cache, if cache
+          information is present in the config JSON.
+      parameters:
+          - name: path
+            type: string
+            description: >
+                Dbus/EEPROM path of the FRU whose keyword needs to be updated.
+          - name: data
+            type:
+                variant[struct[string,string,array[byte]],
+                struct[string,array[byte]]]
+            description: >
+                Information required to update the value of a keyword and the
+                data to write.
+      returns:
+          - name: bytesWritten
+            type: int64
+            description: >
+                Returns -1 on failure otherwise, returns the number of bytes
+                written.
+      errors:
+          - xyz.openbmc_project.Common.Error.InvalidArgument
+
+    - name: ReadKeyword
+      description: >
+          Method to read VPD keyword value. Keyword can be read from hardware or
+          from Dbus.
```

Comment:
Keyword should be read from hardware or DBus based on what?
Description should be such that any lay man should understand how to use the API.

Reply:
updated